### ISSUE TITLE
Run multiple times for one experiment

### ISSUE THREAD

**Issue by keiohta** [keiohta/tf2rl#42]
- We need to evaluate an agent with multiple random seeds (more than 5) to statistically show the performance of the agent, so enable to run multiple times with different random seeds.
  - REF: [How Many Random Seeds? Statistical Power Analysis in Deep Reinforcement Learning Experiments](https://arxiv.org/abs/1806.08295)

**Comment by cambel** [comment]
is there a way to set the desired random seed in tf2rl?

**Comment by keiohta** [comment]
No, currently it is not supported. The main reason I haven't done it (although it is technically not difficult) is TensorFlow does not reproduce results when we use GPUs.
